keyboards/sets3n/zx60/zx60_via_multi_layout.json

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
This file doesn't belong here as it is not used by QMK Firmware at all. Since it is a VIA json, it should be PRed to the `the-via` keyboards repo. instead.

Reply:
Thank you for your patience and guidance